src/bookinfo.ptx

```diff
@@ -154,13 +154,15 @@
     \colorlet{thirdcolor}{gray}
     \colorlet{fourthcolor}{green!50!black}
     \colorlet{fifthcolor}{cyan!70!blue}
+    \colorlet{highlightcolor}{magenta}
 
     % curve, dot, and graph custom styles
     \pgfplotsset{primarycurve/.style   = {color=firstcolor,  mark=none, line width=1pt, {Kite}-{Kite}, solid}}         % primary style for curves
     \pgfplotsset{secondarycurve/.style = {color=secondcolor, mark=none, line width=1pt, {Kite}-{Kite}, dashdotted}}    % secondary style for curves
     \pgfplotsset{tertiarycurve/.style  = {color=thirdcolor,  mark=none, line width=1pt, {Kite}-{Kite}, dashdotdotted}} % tertiary style for curves
     \pgfplotsset{quaternarycurve/.style= {color=fourthcolor, mark=none, line width=1pt, {Kite}-{Kite}, dotted}}        % quaternary style for curves
     \pgfplotsset{fifthcurve/.style     = {color=fifthcolor,  mark=none, line width=1pt, {Kite}-{Kite}, dashed}}        % fifth style for curves
+    \pgfplotsset{highlightcurve/.style  = {color=highlightcolor,  mark=none, line width=4pt, -, opacity=0.4}}        % fifth style for curves
```

Comment:
I'm going to merge this. But two things when you get a chance. Can you remove the extra "fifth style for curves" comment? Also, we can't solely rely on color for highlighting. Not just for accessibility, but it all comes out in B&W in the print book. In other places, I edited the wording to refer to the "solid" part of the plot, and locally made the other portions be dotted. So when using this, there may be other considerations depending on what wording is used in the surrounding paragraphs.